src/sentry/event_manager.py

```diff
@@ -2475,7 +2477,7 @@ def save_attachments(cache_key: Optional[str], attachments: list[Attachment], jo
 
 
 @metrics.wraps("event_manager.save_transactions.materialize_event_metrics")
-def _materialize_event_metrics(jobs: Sequence[Job]) -> None:
+def _materialize_event_metrics(jobs: Sequence[Job], app_feature: str) -> None:
```

Comment:
I think this may not be the most appropriate method to record this. 
This seems the closest place: https://github.com/getsentry/sentry/blob/master/src/sentry/event_manager.py#L1339
Unless you want to get into the weeds of EventManager and the Node model.